### ISSUE TITLE
Icon naming different to PBO icons

### ISSUE THREAD

**Issue by DE-Schmitt** [jetelain/cTab#35]
The icons of the webapp are named totally different than the icons of the PBO addon. In the webapp you use names like "/img/preview/10061000001211020000.png" while in the addon scripts you use much better readable file names like "o_infantry.paa". I suggest same naming for webapp like for addon. Thus for example o_infantry.png for web and o_infantry.paa for Addon.

**Comment by jetelain** [comment]
Hi,
Icons are named according to NATO APP-6D. I've choosed this naming to make it more convenient to generate images with milsymbol library, it avoid having mapping for icons.
Be aware : icons in  `/img/preview/` are used only for marker menu. Markers on map are generated on the fly by milsymbol  library.

**Comment by DE-Schmitt** [comment]
Since when? In the version I use in the file fnc_translateUserMarker.sqf you simply translate the icon id (e.g. "2") to a specific PAA file which is in the arma or in the ctab directory. I dont see a milsymbol library autogeneration whatever magic there?! Example: 
`if (_markerIcon == 2) exitWith {"\A3\ui_f\data\map\markers\nato\o_motor_inf.paa"};`

**Comment by jetelain** [comment]
Hi, it's in the cTabIRL base code.

The following file compute APP-6D code for markers (rendering is done later on JavaScript side) :
https://github.com/jetelain/cTab/blob/master/cTabIRL/cTabWebApp/Hub/CTabMarkers.cs
```cs
        public static string GetMilSymbol(string iconA, string iconB)
        {
            if (iconA.StartsWith("\\cTab\\img\\mil\\", StringComparison.OrdinalIgnoreCase))
            {
                return Path.GetFileNameWithoutExtension(iconA).PadRight(20, '0');
            }
```
For pre-existing markers, I've kept original names to avoid breaking missions/mods, so there is also a mapping for those files.

**Comment by DE-Schmitt** [comment]
> For pre-existing markers, I've kept original names to avoid breaking missions/mods, so there is also a mapping for those files.

Do I get it right, that if I integrate new icons to the mod addon, I simply have to name them with a magic number that represents a valid APP-6D code and magically the cTabIRL app will be able to show the same icon on the web?

But still the menu of the cTabIRL app is now useful for my personal needs. Thus either way I have to make a fork of cTabIRL. Thus we have to host the app on another server. And thus we can also use readable names for the icon image files. There seems to be no big advantage to use the APP-6D codes when the menu has to be adjusted. And the added menu images also need some valid images. Much to do.

**Comment by jetelain** [comment]
Yes, if you marker image names follow the pattern `\cTab\img\mil\APP6Dcode.app`, cTabIRL will be able to show them on the web.

Current menu customization needs a lot of extra work, it has not been designed to be extensible. As I replied in #45, I've planned to change that.

**Comment by DE-Schmitt** [comment]
Of course web app and ingame menu must be synchronized for a logic user experience. It is quite easy to understand your c# code. Thus its easy to adjust the webapp to our needs.
